### ISSUE TITLE
Livereactload with live css and html

### ISSUE THREAD

**Issue by elclanrs** [mattdesl/budo#132]
I'm trying to get React working with livereactload, and it works good so far, but I can't seem to have css and html reloading at the same time. Doing the following makes livereactload work when I save js files, but no realoading happens when I save css and html files:

```
budo src/app.js:bundle.js --dir app --open -- -p livereactload
```

I tried restricting `--live` to css and html, and it works, but then livereactload doesn't work, the page does a full reload when I save a js file:

```
budo src/app.js:bundle.js --dir app --open --live=*.{html,css} -- -p livereactload
```

I tried in Chrome and Firefox, same behavior. Any ideas?

**Comment by mattdesl** [comment]
Should be fixed now in `8.0.3`.

**Comment by elclanrs** [comment]
Great! It works.
